examples/rpt/count_calculation/rpt_count_calculation.prm

```diff
@@ -0,0 +1,37 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# RPT Monte Carlo technique
+#---------------------------------------------------
+subsection rpt parameters
+    set particle positions file          = positions1.particle
+    set export counts                    = true
+    set iteration number                 = 1000000
+    set random number seed               = auto
+    set reactor radius       			 = 0.1
+    set peak-to-total ratio  			 = 0.4
+    set sampling time        			 = 1
+end
+	
+# --------------------------------------------------
+# Initial value of parameters for tuning
+#---------------------------------------------------
+subsection parameter tuning
+	set tuning                           = false	
```

Comment:
What is this option for?

Reply:
It's useless right now, I did it prior adding the tuning part.

Reply:
Good :)! Was just wondering.